src/WebJobs.Extensions.EasyTables/EasyTableAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace WebJobs.Extensions.EasyTables
```

Comment:
You want to use namespace Microsoft.Azure.WebJobs so these attributes light up automatically when the extension reference is added. See all the other extension attributes. Basically you want to ensure that intellisense shows your attribute w/o the user having to know the namespace.